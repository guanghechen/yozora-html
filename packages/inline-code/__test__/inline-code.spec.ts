import { InlineCodeType } from '@yozora/ast'
import renderInlineCode from '../src'

describe('snapshot', function () {
  it('basic', function () {
    expect(
      renderInlineCode({
        type: InlineCodeType,
        value: 'dijkstra'
      }),
    ).toMatchSnapshot()
  })
})
