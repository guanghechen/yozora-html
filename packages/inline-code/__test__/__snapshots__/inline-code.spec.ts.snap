// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot basic 1`] = `"<code class="yozora-inline-code">dijkstra</code>"`;
